lib/order_form.rb

```diff
@@ -0,0 +1,32 @@
+class OrderForm
+  
+  attr_reader :order
+
+  def initialize
+    @order = []
+    @dishes = { Noodles: 3.00, Rice: 2.50 }
+  end
+
+  def show_dishes
+    @dishes.map{ |k,v| "#{k} (£#{'%.2f' % v})"}.join(', ')
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space missing inside }.